pyproject.toml

```diff
@@ -1,12 +1,27 @@
 [tool.poetry]
-name = "python_project"
+name = "tad"
 version = "0.1.0"
 description = ""
-authors = ["ai-validation-team <ai-validation@minibzk.nl>"]
+authors = ["ai-validatie-team <ai-validatie@minbzk.nl>"]
 readme = "README.md"
+license = "EUPL-1.2"
+repository = "https://github.com/MinBZK/tad"
+keywords = ["AI", "Validation", "Transparency", "Algorithm"]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Framework :: FastAPI",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Programming Language :: Python :: 3",
+    "Topic :: Scientific/Engineering :: Artificial Intelligence",
+    "Typing :: Typed"
+]
+packages = [
+    { include = "tad" }
+]
+
 
 [tool.poetry.dependencies]
-python = "^3.11"
+python = "^3.10"
```

Comment:
Why is 3.10 chosen as python version for TAD?

Reply:
It is not, is minimal 3.10

Reply:
But was there a specific reason to decrease the number compared to the python project template?

Reply:
If we want to support several version of python you need to specify the lowest number that you support else poetry complains. As you can see in the test of the ci/cd the project currenlty supports 3.10, 3.11 and 3.12.

Reply:
We specify here ^3.10. this means every version higher than 3.10. So poetry can install 3.10 but also 3.12 if it wants to

Reply:
See here for more information about the carrot: https://python-poetry.org/docs/dependency-specification/#version-constraints

Reply:
Okay cool, thanks for explaining :)